concourse/client.py

```diff
@@ -179,23 +188,72 @@ def build_plan(self, build_id):
         return self._api_url('builds', str(build_id), 'plan', prefix_team=False)
 
 
-class ConcourseApi(object):
+class ConcourseApiRoutesV3(ConcourseApiRoutesBase):
+    '''
+    Implemets Concourse version 3 specific api route methods
+
+    Not intended to be used outside of this module. Instantiated by factory method 'from_cfg'
+    '''
+    def __init__(self, base_url: str, team: str, api_suffix=CONCOURSE_API_SUFFIX):
```

Comment:
why duplicate the c'tor? looks identical between v3 and v4

Reply:
Done